tool/tctl/sso/configure/github.go

```diff
@@ -0,0 +1,145 @@
+// Copyright 2022 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package configure
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/gravitational/teleport/api/types"
+	"github.com/gravitational/teleport/lib/auth"
+	"github.com/gravitational/teleport/lib/utils"
+
+	"github.com/gravitational/kingpin"
+	"github.com/gravitational/trace"
+)
+
+type ghExtraFlags struct {
+	connectorName      string
+	ignoreMissingRoles bool
+}
+
+func addGithubCommand(cmd *SSOConfigureCommand) *AuthKindCommand {
+	spec := types.GithubConnectorSpecV3{}
+
+	gh := &ghExtraFlags{}
+
+	sub := cmd.ConfigureCmd.Command("github", "Configure Github auth connector.")
+	// commonly used flags
+	sub.Flag("name", "Connector name.").Default("github").Short('n').StringVar(&gh.connectorName)
+	sub.Flag("teams-to-logins", "Sets teams-to-logins mapping in the form 'organization,team_name,role1,role2,...'. Repeatable.").
+		Short('r').
+		Required().
+		PlaceHolder("org,team,role1,role2,...").
+		SetValue(newTeamsToLoginsParser(&spec.TeamsToLogins))
+	sub.Flag("display", "Display controls how this connector is displayed.").StringVar(&spec.Display)
+	sub.Flag("id", "Github app client ID.").PlaceHolder("ID").Required().StringVar(&spec.ClientID)
+	sub.Flag("secret", "Github app client secret.").Required().PlaceHolder("SECRET").StringVar(&spec.ClientSecret)
+
+	// auto
+	sub.Flag("redirect-url", "Authorization callback URL.").PlaceHolder("URL").StringVar(&spec.RedirectURL)
+
+	// ignores
+	ignoreMissingRoles := false
+	sub.Flag("ignore-missing-roles", "Ignore non-existing roles referenced in --claims-to-roles.").BoolVar(&ignoreMissingRoles)
```

Comment:
I don't see where `ignoreMissingRoles` is used after `.BoolVar(&ignoreMissingRoles)` parse 

I think taht `&gh.ignoreMissingRoles` flag should be used there instead of a local  `ignoreMissingRoles := false`  var
```suggestion
	sub.Flag("ignore-missing-roles", "Ignore non-existing roles referenced in --claims-to-roles.").BoolVar(&gh.ignoreMissingRoles)
```

Reply:
Good catch, it is as you say.

Reply:
Fixed.